### PR TITLE
Vue3 migration & environment update

### PR DESCRIPTION
**Pull request by 0xhardrefresh**

### CODE REVIEW

**Review by 0xidkfa — DISMISSED** [review]
LGTM. Romy and I did some testing on the front-end with Tenderly forks. Everything appears to be working.

**Review by 0xm3rlin — COMMENTED** [review]
Tentative approval, pls have another check @0xidkfa 
great to see cypress and vite added, currently there is no test for vite?

**Comment by 0xhardrefresh** [comment]
> Tentative approval, pls have another check @0xidkfa great to see cypress and vite added, currently there is no test for vite?

there is already support for testing, namely added [vitest](https://vitest.dev/)

File: src/utils/collateralConfig/cvxRen/depositConfig.js [hunk 1]
```diff
@@ -1,17 +1,21 @@
 import depositTokenAbi from "@/utils/abi/tokensAbi/crvRenWBTC";
 import mainTokenAbi from "@/utils/abi/tokensAbi/cvxRen";
 
+const useImage = ((url) => {
```
Comment (0xidkfa) [inline src/utils/collateralConfig/cvxRen/depositConfig.js:4]: Should we refactor this out into a common file? Can probably do in a v2...

File: src/components/myPositions/SpecPosBorrowItem.vue [hunk 2]
```diff
@@ -243,19 +243,30 @@ export default {
...
           title: "Required Drop in price",
           tooltipText:
             "If your Collateral Price drops by this amount, you will be flagged for liquidation",
-          value: Vue.filter("formatUSD")(this.minPrice),
+          value: filters.formatUSD(this.minPrice),
         },
       ];
     },
   },
+  methods: {
```
Comment (0xidkfa) [inline src/components/myPositions/SpecPosBorrowItem.vue:259]: Is there a way to mix this in? Noticed this is used in a ton of places...

**Comment by 0xidkfa** [comment]
Left a couple of comments, but none are urgent fixes. LGTM!

**Review by 0xidkfa — APPROVED** [review]

**Review by 0xm3rlin — APPROVED** [review]

**Pull request merged**